### DISCUSSION TITLE
[Script request] Checkmk IT monitoring tool

### DISCUSSION THREAD

**Discussion by arkdev1** [community-scripts/ProxmoxVE#398] [Request script]
### Application Name

Checkmk

### Website

https://checkmk.com/

### Description

Effective IT monitoring and analytics are critical for maintaining optimal performance and ensuring uninterrupted operations in our IT infrastructure. I think that it can be put in the "Monitoring & Analytics" section.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment]
Added.
